sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -450,10 +451,11 @@ template <typename T, uint8_t NElts = 1,
           typename AccessorTy>
 __ESIMD_API std::enable_if_t<!std::is_pointer<AccessorTy>::value,
                              __ESIMD_NS::simd<T, NElts>>
-lsc_block_load(AccessorTy acc, uint32_t offset) {
+lsc_block_load(AccessorTy acc, uint32_t offset,
+               __ESIMD_NS::simd_mask<1> pred = 1) {
 #ifdef __ESIMD_FORCE_STATELESS_MEM
   return lsc_block_load<T, NElts, DS, L1H, L3H>(
-      __ESIMD_DNS::accessorToPointer<T>(acc, offset));
+      __ESIMD_DNS::accessorToPointer<T>(acc, offset, pred));
```

Comment:
`pred` should apparently be an argument of `lsc_block_load`, not `accessorToPointer`.
This should have been caught by the https://github.com/intel/llvm-test-suite/pull/1194 test, right? Do you have idea why it wasn't? Probably, the test should also compile/run with forced stateless memory access